tests/libmigration/migration.go

```diff
@@ -202,21 +202,20 @@ func ClearDedicatedMigrationNetwork() *v1.KubeVirt {
 func GenerateMigrationCNINetworkAttachmentDefinition() *k8snetworkplumbingwgv1.NetworkAttachmentDefinition {
 	nad := &k8snetworkplumbingwgv1.NetworkAttachmentDefinition{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "migration-cni",
+			Name:      "migration-l2-network",
 			Namespace: flags.KubeVirtInstallNamespace,
 		},
 		Spec: k8snetworkplumbingwgv1.NetworkAttachmentDefinitionSpec{
 			Config: `{
-      "cniVersion": "0.3.1",
-      "name": "migration-bridge",
-      "type": "macvlan",
-      "master": "` + flags.MigrationNetworkNIC + `",
-      "mode": "bridge",
-      "ipam": {
-        "type": "whereabouts",
-        "range": "172.21.42.0/24"
-      }
-}`,
+				"cniVersion": "0.3.1",
+				"name": "migration-l2-network",
+				"type": "ovn-k8s-cni-overlay",
```

Comment:
We do not cover this CNI in k/kubevirt e2e tests.
Even the use of the `macvlan` CNI plugin is non-standard, but for the scenario here it is reasonable.